pkg/agent/agent.go

```diff
@@ -556,6 +625,10 @@ func (a *Agent) updateConfig(cfg metav1.Object) {
 	}
 
 	a.currentCfg = cfg
+
+	if err := a.configure(cfg); err != nil {
+		log.Warn("failed to reconfigure agent: %v", err)
+	}
```

Comment:
Isn't this unreachable code?
[setupClinets()](https://github.com/containers/nri-plugins/blob/72bfdfcf4cf0998449e327892075a2d570ae82c5/pkg/agent/agent.go#L183C11-L183C23) returns an error within the `Start()` before we reach `updateNodeConfig()`.